api/src/test/java/org/openmrs/notification/db/hibernate/HibernateAlertDAOTest.java

```diff
@@ -9,34 +9,65 @@
  */
 package org.openmrs.notification.db.hibernate;
 
+import org.junit.Assert;
+
 import org.junit.Before;
 import org.junit.Test;
+import org.openmrs.User;
 import org.openmrs.api.context.Context;
-import org.openmrs.notification.AlertService;
+import org.openmrs.notification.Alert;
 import org.openmrs.test.BaseContextSensitiveTest;
+import org.springframework.beans.factory.annotation.Autowired;
 
 /**
  * This class tests the hibernate alert data access. TODO Consider changing this and all subsequent
  * tests to use dbunit
  */
 public class HibernateAlertDAOTest extends BaseContextSensitiveTest {
 	
+	private static final String DATA_XML = "org/openmrs/api/db/hibernate/include/HibernateAlertDAOTestDataSet.xml";
+	
+	@Autowired
+	private HibernateAlertDAO hibernateAlertDAO;
+	
 	@Before
-	public void runBeforeEachTest() {
-		authenticate();
+	public void setUp() {
+		executeDataSet(DATA_XML);
+	}
+
+	@Test
+	public void saveAlert_shouldSaveAlertToDb() {
+		Alert alert=new Alert();
+		alert.setText("Coding time");
+		alert.setId(5);
+	    hibernateAlertDAO.saveAlert(alert);
+	    Assert.assertNull(Context.getAlertService().getAlert(5));
```

Comment:
@ibacher Hi ian,thanks for the reviews.I made the recommended changes here.cc @dkayiwa